integration_tests/testsuite/driver.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+
+import argparse
+import logging
+import sys
+
+import deploy_app
+import test_root
+import test_util
+import test_monitoring
+import test_logging
+
+# TODO (nkubala): make this a configurable parameter from cloudbuild
+# required to be paired with '--no-deploy'
+DEFAULT_URL = "https://nick-cloudbuild.appspot.com"
+
+def _main():
+  logging.getLogger().setLevel(logging.DEBUG)
+
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--image', '-i', help='Newly-constructed base image to build sample app on')
+
+  parser.add_argument('--directory', '-d', help='Root directory of sample app')
+  # TODO (nkubala): potentially add support for multiple app directories to deploy
+  # parser.add_argument('--directory', '-d', help='Root directory of sample app', action='append')
+  parser.add_argument('--no-deploy', action='store_true', help='Flag to skip deployment of app (must provide app URL)')
```

Comment:
For the initial cut we should probably make this all --no-deploy. Adding in deployment will complicate this initiali version quite a bit.